index.html

```diff
@@ -1,231 +1,298 @@
-                    </li>
-                </ul>
-                <div class="button-resume">
-                    <button type="button" class="btn">
-                        Get my resume
-                    </button>
-                </div>
             </div>
-            <div class="right">
-                <div class="lang">
+            <ul class="language-list">
+                <li class="lang">
+                    <p>
+                        Languages
+                    </p>
+                    <img class="imglang" src="images/arrowdown.png" alt="arrow">
+                </li>
+                <li class="lang-skill">
+                    <img src="images/js.png" alt="js">
+                    <p>
+                        JavaScript
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="images/css.png" alt="html">
+                    <p>
+                        CSS
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="images/html.png" alt="css">
+                    <p>
+                        HTML
+                    </p>
+                </li>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="images/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="images/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
+    <!--Add contact section-->
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h1 class="contact-title">Contact me</h1>
+            <p class="contact-text">
+                If you have an application you are interested in developing,
+                a feature
+                that you need built or a project that needs coding. I’d love
+                to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
+        </div>
+    </section>
 
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
- [ ] [OPTIONAL] Great job on your `contact form` however your `contact form` and `about section` are repeating on your portfolio. This is probably caused by the conflicting code between branches that you might have tried to solve. I would suggest you  to remove all the code from line `294 - 371` and then removing line `374` as well


https://user-images.githubusercontent.com/44798044/229990444-ee33f3f2-7ad5-49e4-ae4f-cbbfef505d46.mov